packages/inferno-component/src/index.ts

```diff
@@ -19,19 +27,20 @@ import VNodeFlags from 'inferno-vnode-flags';
 let noOp = ERROR_MSG;
 
 if (process.env.NODE_ENV !== 'production') {
-	noOp = 'Inferno Error: Can only update a mounted or mounting component. This usually means you called setState() or forceUpdate() on an unmounted component. This is a no-op.';
+	noOp =
+		'Inferno Error: Can only update a mounted or mounting component. This usually means you called setState() or forceUpdate() on an unmounted component. This is a no-op.';
 }
 
 const componentCallbackQueue: Map<any, Function[]> = new Map();
 
 export interface ComponentLifecycle<P, S> {
-	componentDidMount?(): void;
```

Comment:
I think that it was just an easy way to get started printing typescript. The current way interface are printed is to be inside of the code that prints objects. But the code likely shouldn't be shared. It is indeed better to print it with semi colons.